### PR TITLE
adding doc with release instructions

### PR DESCRIPTION
**Pull request by SaOgaz**
Pretty brief at the moment... can always be filled out more later.

### CODE REVIEW

**Comment by nmearl** [comment]
Can you create a new toc under a `Developer Documentation` heading to distinguish it from user documentation?

**Comment by SaOgaz** [comment]
Zenodo errors?

File: docs/releasing.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.. highlight:: shell
+
+====================
+Release Instructions
+====================
+
+You will need to set up a gpg key, PyPI account, and install twine before
```
Comment (eteq) [inline docs/releasing.rst]:
```suggestion
You will need to set up a gpg key (see the `astropy docs section on this <http://docs.astropy.org/en/stable/development/releasing.html#key-signing-info>`_ for more), PyPI account, and install twine before
```

**Comment by SaOgaz** [comment]
LGTM, thanks for the edits @eteq!

**Pull request merged**